resources/sql_scripts/update_scripts/firmware_update.sql

```diff
@@ -0,0 +1,75 @@
+-- Run this SQL update script if you already have a deployed CV Manager PostgreSQL database
+-- This file will recreate the firmware tables, make sure to backup any data that may be located
+-- in the firmware table before running this.
+
+ALTER TABLE public.rsus
+    DROP COLUMN os_version,
+    DROP COLUMN firmware_version;
+
+DROP TABLE public.firmware_upgrade_rules;

```

Comment:
Are we making the assumption here that no firmware upgrade rules were in place that we'd need to reinsert?

Reply:
Yeah, I am fairly confident no one is using this feature since it wasn't even fully functional before.